lib/services/local/workload_identity_test.go

```diff
@@ -353,3 +353,41 @@ func TestWorkloadIdentityService_UpdateWorkloadIdentity(t *testing.T) {
 		require.Error(t, err)
 	})
 }
+
+func TestWorkloadIdentityParser(t *testing.T) {
```

Comment:
nit: add `t.Parallel()` here and within each t.Run?

Reply:
f4d1efcc2b